### DISCUSSION TITLE
Is there a way use selenium and work with the driver from python?

### DISCUSSION THREAD

**Discussion by y-vas** [vividus-framework/vividus#4125] [Q&A]
We want to use selenium and add some customs steps on runtime with python?
Like streaming the browser on realtime, switch between tabs and extract data at the same time from one tab and check if its the same in another.

**Comment by valfirst** [comment]
VIVIDUS is based on Java, so any custom steps should be implemented on JVM-based language: Java, Kotlin, Groovy, Scala. It's not possible to create own steps implemented using Python.
